editor/views/overview.py

```diff
@@ -1,29 +1,47 @@
+import os
+
 import streamlit as st
 
-from core.state import CurrentStep
 from core.state import Metadata
+import mlcroissant as mlc
 from utils import needed_field
-from utils import set_form_step
 
 
 def render_overview():
     metadata = st.session_state[Metadata]
-    name = st.text_input(
-        label=needed_field("Name"),
-        value=metadata.name,
-        placeholder="Dataset",
-    )
-    description = st.text_area(
-        label="Description",
-        value=metadata.description,
-        placeholder="Provide a clear description of the dataset.",
-    )
-    st.subheader(f"{len(metadata.distribution)} Resources")
-    st.subheader(f"{len(metadata.record_sets)} Record Sets")
-    st.session_state[Metadata].update_metadata(
-        name=name,
-        description=description,
-        license=metadata.license,
-        url=metadata.url,
-        citation=metadata.citation,
-    )
+    col1, col2 = st.columns([1, 1], gap="medium")
+    with col1:
+        name = st.text_input(
+            label=needed_field("Name"),
+            value=metadata.name,
+            placeholder="Dataset",
+        )
+        description = st.text_area(
+            label="Description",
+            value=metadata.description,
+            placeholder="Provide a clear description of the dataset.",
+        )
+
+        st.subheader(f"{len(metadata.distribution)} Files")
+        st.subheader(f"{len(metadata.record_sets)} Record Sets")
+        metadata.update_metadata(
+            name=name,
+            description=description,
+            license=metadata.license,
+            url=metadata.url,
+            citation=metadata.citation,
+        )
+    with col2:
+        st.header("Croissant File Validation")
+        try:
+            issues = metadata.to_canonical().issues
+            if issues.errors or issues.warnings:
+                st.subheader("Errors:")
+                st.text(issues.errors)
```

Comment:
Should you do two `if` blocks (one for errors, one for warnings)?

Otherwise, you'll see the empty set: 
<img width="556" alt="image" src="https://github.com/mlcommons/croissant/assets/17081356/6b832380-51ef-49af-a515-e3b852eabee1">